src/ol/style/expressions.js

```diff
@@ -784,3 +785,58 @@ Operators['case'] = {
     return result;
   },
 };
+Operators['in'] = {
+  getReturnType: function (args) {
+    return ValueTypes.BOOLEAN;
+  },
+  toGlsl: function (context, args) {
+    assertArgsCount(args, 2);
+
+    const haystack = [];
+    const needle = [];
+    let result;
+    if (getValueType(args[1]) & ValueTypes.NUMBER_ARRAY) {
+      const keyword = expressionToGlsl(context, args[0]);
+      const input = args[1];
+      assertNumbers(input);
+      let i = 0;
+      let inputArr = ['array'];
+      // @ts-ignore
```

Comment:
Are these `@ts-ignore` necessary?

Reply:
I had them there because, if I don't include them, my IDE marks the `input.includes()` and the `input.forEach()` as errors.  It's because `input` can be one of a few different types.  The `if` statements and `assert` statements should insure that those functions are never invoked on a type that doesn't support them, and the code still runs fine without the `@ts-ignore`.

Is there a cleaner way to handle those errors?